### DISCUSSION TITLE
MEPLHAA Scripts for heater

### DISCUSSION THREAD

**Discussion by silvious22** [RavenSystem/esp-homekit-devices#2074] [MEPLHAA Scripts]
Hello! Please, help me with migration to Merlin

{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12}]},"1":{"a":0},"3":{"r":[{"g":12,"v":1}]},"5":{"a":0},"t":21,"s":5,"g":14,"b":[],"n":4,"j":10,"w":1,"m":10,"x":75,"d":5},{"t":24,"b":[],"n":4,"g":14,"j":10}]}

**Comment by pasparakis** [comment]
{"c":{"l":13,"b":[[0,5]],"io":[[[13],3],[[0,14],6],[[12],2]]},"a":[{"0":{"r":[[12]]},"1":{"a":0},"3":{"r":[[12,1]]},"5":{"a":0},"t":21,"s":5,"g":14,"b":[],"n":4,"j":10,"w":1,"m":10,"x":75,"d":5},{"t":24,"b":[],"n":4,"g":14,"j":10}]}

I am not sure...try it

**Reply by silvious22** [reply]
Yes, it works, but the installation mode lasts 5-6 seconds, I don’t have time to change wi fi. How to fix it?

**Reply by silvious22** [reply]
and button dont work
